modules/icon/css/README.md

```diff
@@ -160,3 +160,24 @@ selector when injecting.
 ```js
 initializeIcons(null, '.wdc-custom-icon');
 ```
+
+# Canvas Kit CSS Icon List
+
```

Comment:
Can we add a description here? And usage below? I don't know if a user will be able to know what this section is for without those two things.

Reply:
yep! just found an example to use